lib/pcap/source.ex

```diff
@@ -0,0 +1,54 @@
+defmodule Membrane.Element.Pcap.Source do
+  use Membrane.Element.Base.Source
+
+  alias Membrane.Buffer
+  alias Membrane.Event.EndOfStream
+
+  def_output_pads output: [
+                    caps: :any
+                  ]
+
+  def_options packet_transformer: [
```

Comment:
For me, something like `buffer_extractor` would be more meaningful but choose what you prefer